src/flint/fmpz.jl

```diff
@@ -1988,6 +1989,18 @@ function combit!(x::fmpz, c::Int)
     ccall((:fmpz_combit, libflint), Nothing, (Ref{fmpz}, Int), x, c)
 end
 
+@doc Markdown.doc"""
+    tstbit(x::fmpz, c::Int)
+
+Test bit $i$ of x (numbered from 0) and return 1 or 0 depending if it is set.
+"""
+function tstbit(x::fmpz, c::Int)
+   if c < 0
+      return 0
```

Comment:
```suggestion
      return false
```
Also maybe some test code for this with @ inferred ?